R.swift/SwiftTypes/Property.swift

```diff
@@ -12,6 +12,26 @@ protocol Property: UsedTypesProvider, CustomStringConvertible {
   var name: String { get }
 }
 
+struct AnyProperty: Property {
```

Comment:
I implemented type-erasure here so that the compiler doesn't complain when putting `Let`s in `[Property]` arrays